nirnroute.html

```diff
@@ -56,35 +56,35 @@
 	</nav>
 </div>
 <!-- end topbar-->
-<div class="guideFrame">
-
-<div id="mapColumn" class="mainPanel" style="width: 67.2%;">
-<div class="mapContainer" id="mapContainer">
-	<div id="mapSettingsContainer" style="display:none">
-		<label title="Shows all Locations that need to be discovered."><input type="checkbox" name="icon_selection" id="button_Location" checked >Locations</label>
-		<label title="Shows where all Nirnroot are in the overworld."><input type="checkbox" name="icon_selection" id="button_Nirnroot"/>Nirnroots</label>
-		<label title="Shows where all Wayshrines are in the overworld."><input type="checkbox" name="icon_selection" id="button_Wayshrine"/>Wayshrines/Heaven Stones</label>
-		<label title="Shows gates that can be seen from each location."><input type="checkbox" name="icon_selection" id="button_NearbyGates"/>Nearby Gates</label>
-		<br/>
-		<label><input type="radio" name="tsp_selection" id="button_tspNone" title="The fastest way to visit each location"/>No TSP</label>
-		<label><input type="radio" name="tsp_selection" id="button_tspLocation" title="The fastest way to visit each location"/>Location TSP</label>
-		<label><input type="radio" name="tsp_selection" id="button_tspNirnroot" title="The fastest way to visit each nirnroot"/>Nirnroot TSP</label>
-		<br/>
-		<label><input type="checkbox" id="mapShowPrediscovered" class="autosetting">Show prediscovered locations</label>
-		<label><input type="checkbox" id="mapShowDistanceCheck" class="autosetting">Show distance between locations</label>
-		<label><input type="checkbox" id="mapShowFormId" class="autosetting">Show formID on map</label>
-	</div>
-	<div id="wrapper_Map" class="mapWrapper">
-		<button id="showSettingsButton" style="position:absolute">Show Settings</button>
+<div class="contentContainer">
+<div id="mapColumn" class="mainPanel" style="width: 66%;">
```

Comment:
If you make the contentContainer have a 0 margin on all sides, setting this to 67% makes it line up perfectly again.

Reply:
makes the nirnroot images be not slightly clipping off the bottom of the page, the margin adjustment just throws off the nirnroot images by a small amount